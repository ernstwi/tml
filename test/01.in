init foo
bar { x < 10 }

foo -> bar
    guard { x >= 5 }
bar -> baz
