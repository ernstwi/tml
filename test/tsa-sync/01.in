init foo

foo -> bar
    sync { a }
