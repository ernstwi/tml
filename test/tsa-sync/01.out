Parse error at FILE "test/tsa-sync/01.in" 4:14: Unexpected '}'. Expected '?' or '!'
